response.go

```diff
@@ -190,6 +196,24 @@ func (resp *Response) decodeBody() (err error) {
 				if bindCount, err = d.DecodeUint64(); err != nil {
 					return err
 				}
+			case KeyVersion:
+				isIdResponse = true
+				if err = d.Decode(&serverIdInfo.version); err != nil {
+					return err
+				}
+			case KeyFeatures:
+				isIdResponse = true
+				if larr, err = d.DecodeArrayLen(); err != nil {
+					return err
+				}
+
+				serverIdInfo.features = make([]Feature, larr)
+				for i := 0; i < larr; i++ {
+					if err = d.Decode(&feature); err != nil {
+						return err
+					}
+					serverIdInfo.features[i] = feature
+				}
```

Comment:
https://www.tarantool.io/en/doc/latest/dev_guide/internals/iproto/requests/#box-protocol-id

KeyVersion + KeyFeatures are required. Should there be an error if something is missing?

Reply:
Added an error